recipes/root/all/conanfile.py

```diff
@@ -0,0 +1,305 @@
+import os
+import shutil
+import stat
+from glob import glob
+
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class PythonOption:
+    OFF = "off"
+    SYSTEM = "system"
+    # in future we may allow the user to specify a version when
+    # libPython is available in Conan Center Index.
+    ALL = [OFF, SYSTEM]
+    DEFAULT = OFF
```

Comment:
This appears to be unused

Reply:
Well spotted! I have removed the unused variable in c4a2736.